src/System.Diagnostics.Process/src/Resources/Strings.resx

```diff
@@ -243,8 +243,11 @@
   <data name="ProcessorAffinityNotSupported" xml:space="preserve">
     <value>Processor affinity for processes or threads is not supported on this platform.</value>
   </data>
-  <data name="ProcessStartIdentityNotSupported" xml:space="preserve">
-    <value>Starting a process with a different identity is not supported on this platform.</value>
+  <data name="ProcessStartWithPasswordAndDomainNotSupported" xml:space="preserve">
```

Comment:
Please check this message. Is it correct?

Reply:
LGTM